manifest.yml

```diff
@@ -1,8 +1,8 @@
 manifest_version: 1
-name: CLIPTextEncoder
+name: CLIP Text Encoder
```

Comment:
lets keep the name exactly as the class executor name

Reply:
Is there any reason for that? Template specifies that this should be a human readable name

Reply:
Okay, but I see the template has the exact same format - will change then